.gitignore

```diff
@@ -1,16 +1,17 @@
 # Freqtrade rules
-config*.json
+
```

Comment:
these changes already exist in stable branch - you can either merge stable in to your branch 

if that does not work - consider cherry-picking your changes to a new branch